exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `message`
+var message = "This is a string";
```

Comment:
try to avoid using `var`; it can cause issues with scoping https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/ (you're not expected to understand what this means currently, just here in case you're interested). Try to always use `const`, then, if the variable needs reassigning, delcare with `let` instead